process/geometry/plasma_geometry.py

```diff
@@ -36,19 +36,16 @@ def plasma_geometry(
     based on the given major radius, minor radius, triangularity, and elongation at 95% of the plasma surface. It also
     considers whether the plasma configuration is single null or double null.
 
-    :param rmajor: Plasma major radius.
-    :param rminor: Plasma minor radius.
-    :param triang: Plasma triangularity at separatrix.
-    :param kappa: Plasma elongation at separatrix.
-    :param i_single_null: Switch for single null (1) or double null (0) plasma configuration.
-    :param i_plasma_shape: Switch for plasma shape (0 for double arc, 1 for Sauter).
-    :param square: Square term for Sauter plasma shape.
-    :return: A dataclass containing the plasma elongation and the radial and vertical coordinates of the plasma.
-
-    The returned PlasmaGeometry dataclass contains:
-    - rs: List of radial coordinates for the inner and outer plasma boundaries.
-    - zs: List of vertical coordinates for the inner and outer plasma boundaries.
-    - kappa: Calculated plasma elongation.
+    :param float rmajor: Plasma major radius.
+    :param float rminor: Plasma minor radius.
+    :param float triang: Plasma triangularity at separatrix.
+    :param float kappa: Plasma elongation at separatrix.
+    :param int i_single_null: Switch for single null (1) or double null (0) plasma configuration.
+    :param int i_plasma_shape: Switch for plasma shape (0 for double arc, 1 for Sauter).
+    :param float square: Square term for Sauter plasma shape.
+    :returns: A dataclass containing the plasma elongation and the radial and vertical coordinates of the plasma.
+    :rtype: PlasmaGeometry
+
```

Comment:
Is this the right form for types (https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html)? I'm not so clued up on this, but I haven't seen this form before.